jbehave-core/src/main/java/org/jbehave/core/configuration/Configuration.java

```diff
@@ -236,6 +242,13 @@ public StoryLoader storyLoader() {
         return storyLoader;
     }
 
+    public ExamplesTableFactory examplesTableFactory() {
+        if (examplesTableFactory == null) {
+            examplesTableFactory = new ExamplesTableFactory(this);
```

Comment:
use constructor that accepts set of parameters, to avoid cyclic dependencies at least here